.vscode/launch.json

```diff
@@ -41,6 +41,16 @@
       "name": "Rust Gen CI",
       "program": "${workspaceFolder}/target/rust/debug/enso-build-ci-gen",
       "args": []
-    }
+    },
+   {
+      "type": "lldb",
+      "request": "launch",
+      "preLaunchTask": "Rust ./run",
+      "name": "Rust ./run",
+      "program": "${workspaceFolder}/target/rust/buildscript/enso-build-cli",
+      // provide arguments, for example: 
+      // "args": ["backend", "stdlib-api-check"]
```

Comment:
With c4174e5d8 it should be easy to debug `./run` executable:

<img width="1211" height="364" alt="debug ./run" src="https://github.com/user-attachments/assets/78906bd4-7436-410f-b94c-f1f2f66cb748" />

Change the `args` list and press F5.